src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -205,17 +205,14 @@ const ray::rpc::ActorDeathCause GcsActorManager::GenNodeDiedCause(
     const std::string ip_address,
     std::shared_ptr<rpc::GcsNodeInfo> node) {
   ray::rpc::ActorDeathCause death_cause;
+
   auto actor_died_error_ctx = death_cause.mutable_actor_died_error_context();
   AddActorInfo(actor, actor_died_error_ctx);
   actor_died_error_ctx->set_error_message(
-      absl::StrCat("The actor is dead because its node has died. Node Id: ",
+      absl::StrCat("The actor died because its node has died. Node Id: ",
                    NodeID::FromBinary(node->node_id()).Hex()));
```

Comment:
I think we should include the node death message here so we don't need to set it in exceptions.py

Reply:
Good call. Updated.